gravitational/gravity
# [5.5.x] system debug endpoint in container

## Description

This PR adds support for debug endpoint inside the Planet container. The endpoint is only query-able from the node locally - using the dedicated `planet debug-status` command.

## Type of change


* New feature (non-breaking change which adds functionality)
* Internal change (not necessarily a bug fix or a new feature)

## Linked tickets and other PRs



* Refs https://github.com/gravitational/gravity/issues/1146
* Refs https://github.com/gravitational/gravity/issues/1091
* Requires https://github.com/gravitational/planet/pull/569
* Requires https://github.com/gravitational/satellite/pull/165

## TODOs



## Implementation


## Testing done

* Install a single-node as well as 3-node clusters.
* Ensure that `gravity status` works as expected
* Ensure that the agent can be manually stopped/restarted
* Ensure that the new command `planet debug-status` reports the goroutine profile as text.
